fpdf/html.py

```diff
@@ -825,6 +977,30 @@ def error(self, message):
         raise RuntimeError(message)
 
 
+def ul_prefix(ul_type):
```

Comment:
No CoerciveEnums for those two?

Reply:
The special case with `if len(ul_type) == 1` makes it difficult to convert this to a `CoerciveEnum`.

But if's deemed preferable, we could still perform this conversion in the future. It's not part of the user-facing functionalities.